Formula/proxsuite.rb

```diff
@@ -0,0 +1,50 @@
+class Proxsuite < Formula
+  desc "Advanced Proximal Optimization Toolbox"
+  homepage "https://github.com/Simple-Robotics/proxsuite"
+  url "https://github.com/Simple-Robotics/proxsuite/releases/download/v0.2.2/proxsuite-0.2.2.tar.gz"
+  sha256 "da72d900c86fe77a1572f4f2866600ef56a63bd72f050206cc0a2c1643c8f185"
+  license "BSD-2-Clause"
+  head "https://github.com/Simple-Robotics/proxsuite.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "pkg-config" => :build
+  depends_on "eigen"
+  depends_on "numpy"
+  depends_on "python@3.10"
+  depends_on "scipy"
+  depends_on "simde"
+
+  def python3
+    Formula["python@3.10"].opt_libexec/"bin/python"
+  end
```

Comment:
Since it's only used once this shouldn't be a function

Reply:
OK. I've handled it.